projects/storefrontapp-e2e-cypress/cypress/helpers/customer-ticketing/customer-ticketing.ts

```diff
@@ -149,3 +149,25 @@ export function verifyTicketDoesNotExist(ticketDetails: TestTicketDetails) {
 export function visitApparelUKTicketListingPage(){
   visitPage('apparel-uk-spa/en/GBP/my-account/support-tickets', 'apparelTicketListingPage');
 }
+
+export function visitTicketDetailsPageForTicketId(ticketId: string) {
+  const ticketDetailsPageUrl = `/my-account/support-ticket/${ticketId}`;
+  visitPage(ticketDetailsPageUrl);
+}
+
+export function visitTicketDetailsPageForTicketInPosition(ticketRowIndex: number){
```

Comment:
I'm not sure why this function is needed

Reply:
I understand the idea and I do need it for sorting but in this PR we have clickTicketFromTicketListing and visitTicketDetailsPageForTicketInPosition

Reply:
Oh my bad, in that we need to choose one.
My personal preference is "click..." because the "visit..."  could be mean we are visiting through URL